### ISSUE TITLE
[BUG] Add more detail to automated checks role

### ISSUE THREAD

**Issue by ferBonnin** [Microsoft/accessibility-insights-windows#186]
**Describe the bug**
From user feedback, emphasize the role of automated checks in the tool: "that no automated tool can find even a majority of accessibility defects on a web page"

**To Reproduce**
1. Open Accessibility Insights for Windows
2. select continue on the What's new section

OR
1. Go to https://accessibilityinsights.io/docs/en/windows/getstarted/fastpass

**Expected behavior**
The Automated checks section of FastPass should clarify that it only finds some of the most common accessibility issues

**Comment by neocln** [comment]
@ferBonnin could you provide a more specific text for the Automated checks section?

**Comment by ferBonnin** [comment]
@neocln adding the text here:
"Did you know?
Automated checks can detect some common accessibility problems, such as missing or invalid properties. However, most accessibility problems can be identified only through manual testing."

**Comment by karanbirsingh** [comment]
@ferBonnin 
- it's not clear where the text should go - can you choose from these options? thanks!
- we also need to know whether updating the website text is part of this issue, and if so, where the text should be placed.

options:
1. in the live inspect instructions seen after clicking 'continue':
![live inspect instructions](https://user-images.githubusercontent.com/7775527/55819089-e71fb380-5aac-11e9-9431-a64547db5fd9.png)

2. in the automated checks pane before tests are run:
![live inspect > tests](https://user-images.githubusercontent.com/7775527/55819165-07e80900-5aad-11e9-805c-da5df70ebbb3.png)

3. in the automated checks pane after tests are run:
![completed tests](https://user-images.githubusercontent.com/7775527/55819208-1c2c0600-5aad-11e9-9595-88a8c70df907.png)

**Comment by ferBonnin** [comment]
@cheade thoughts on where to add this text?

I would prefer 2 (given that 3 will look too busy if we add that text)

**Comment by cheade** [comment]
I think #2 is the place where it would make most sense. However there is already a good chunk of text on the page and I have a guess that many users will gloss over this piece of information. 

Is there enough specific feedback that it has to be in the tool itself and not just within the link to "Learn more about automated checks"?

**Comment by ferBonnin** [comment]
@cheade yes, the ask was to add it in both tool and documentation.
